### DISCUSSION TITLE
Inquiry 8

### DISCUSSION THREAD

**Discussion by yujispin990** [winstxnhdw/lc-hax#171] [Ideas]
Is it possible to permanently turn off the tv? or maybe as opposed to my previous suggestion of making everyone hear me, is it also possible to make it so that we play an audio that he have preset so everyone could hear?

**Comment by winstxnhdw** [comment]
So the reason why audio is taking so long is because they use a third-party voice-chat library called `Dissonance`, which I don't fully understand yet. All of what you mentioned might be possible but I'll only really know after investigating deeply.

**Reply by yujispin990** [reply]
> So the reason why audio is taking so long is because they use a third-party voice-chat library called `Dissonance`, which I don't fully understand yet. All of what you mentioned might be possible but I'll only really know after investigating deeply.

Ah! understood, but there might be a misunderstanding, i guess it looked like i am impatient 😅 , i was actually just curious if you could just have a folder that you place your audio and be able to just play it with out using any other third party software. Of course, I understand things like these are not something you can snap your fingers and make just like that. I apologize if i had offended you in any way.

**Reply by winstxnhdw** [reply]
Hmm.. That sounds more like a job for soundboards tho?

**Reply by yujispin990** [reply]
> Hmm.. That sounds more like a job for soundboards tho?

Actually that was the plan all along 😆. I wanted to play lets say a creepy soundboard audio in the corridor, if there was an "Everyone hears me" feature i could make it so that i just stay in the ship and scare people anywhere in the map XD.

**Reply by winstxnhdw** [reply]
Let me try to get a everyone can hear me feature.

**Reply by yujispin990** [reply]
Anti-kick invisibility and radar blocking are just the best! Yesterday, I hopped into a lobby, and they didn't even notice I was there. I communicated only using /signal, and they thought there was a divine entity on the ship giving them stuff XD. I wish I had recorded; it was hilarious!

**Reply by yujispin990** [reply]
> Let me try to get a everyone can hear me feature.


Aww thank you so much, you dont need to rush though, if there are other things that are much more important, please pursue those matters first.
